### ISSUE TITLE
Include ethernet packets

### ISSUE THREAD

**Issue by mfaerevaag** [joswr1ght/nm2lp#2]
Is there any way of including all traffic, and not only wireless?

Or does this require extensive code changes?

EDIT:
After commenting out the `continue` statement, including everything, I see that the different offsets are missing, here: https://github.com/joswr1ght/nm2lp/blob/master/nm2lp.c#L111-L122
Where can these offsets be found?

**Comment by joswr1ght** [comment]
You don't need `nm2lp` to convert the packet capture if it is just Ethernet -- you can do it with `editcap`:

```
$ editcap -F pcap -T ether sample-netmon-ethernet.cap sample-netmon-ethernet.pcap
```

**Comment by joswr1ght** [comment]
The offsets in https://github.com/joswr1ght/nm2lp/blob/master/nm2lp.c#L111-L122 are specific to the wireless link type being used, and then for the options present in the DLT header (for example, DLT_PPI and DLT_IEEE802_11_RADIO have variable header length, requiring parsing of those headers to determine the offset. What's more, there is no guarantee that the header length will be consistent from frame-to-frame (though it usually is), requiring per-packet calculations for correct offset identification).

This is why I stopped working on this. I made it work as a one-off for what I needed, and figured I'd update it as needed in the future. 🤣
